docs/src/PolyhedralGeometry/pg_cones.md

```diff
@@ -12,6 +12,36 @@ Pages = ["pg_cones.md"]
 
 # Cones
 
+
 ## Introduction
 
-## Constructions
+Let $\mathbb{F}$ be an ordered field; the most prominent case here is where $\mathbb{F}=\mathbb{Q}$ are the rational numbers.
+
+A set $C \subseteq \mathbb{F}^n$ is called a (polyhedral) cone if it can be written as the set of nonnegative linear combinations of finitely many vectors in $\mathbb{F}^n$.
+Equivalently, cones can be written as the intersection of finitely many homogeneous linear inequalities.
+
+Any cone is a special case of a polyhedron.
+Conversely, intersecting a cone with a suitable affine hyperplane yields a polyhedron whose faces are in bijection with the faces of the cone.
+Going back and forth between polyhedra and their homogenizations, the cones, is a frequent operation.
+This is one reason for keeping cones as a distinct type.
```

Comment:
That doesn't really make it clearer to me. I am not doubting there are good reasons, it's just this sentence doesn't make it clear to me; so I'd say either elaborate, or else ditch this?